src/core/instanceManager.js

```diff
@@ -117,6 +117,14 @@ module.exports = class InstanceManager {
 
         return this.metadataManager.resolveMetaFeature(type, feature)
             .then(meta => {
+                if (meta.ttl === 0) {
```

Comment:
What happens if the ttl is negative?

Reply:
Cannot happen, since all incoming data and the types definition will be checked using the accompanied JSON Schema

Reply:
Ok but it depends on the outside, right? To make it more robust and independent that could be small fix (with no overhead) if the behaviour for negative values would be the same.